lavalink/lavalink.py

```diff
@@ -196,7 +196,12 @@ def _get_event_args(data: enums.LavalinkEvents, raw_data: dict):
     if data == enums.LavalinkEvents.TRACK_END:
         extra = enums.TrackEndReason(raw_data.get("reason"))
     elif data == enums.LavalinkEvents.TRACK_EXCEPTION:
-        extra = raw_data.get("error")
+        extra = {
+            "message": raw_data.get("exception", {}).get(
+                "message", "Something went wrong when decoding the track."
+            ),
+            "cause": raw_data.get("exception", {}).get("cause", "Unhandled Exception"),
+        }
```

Comment:
All of the keys are always present:
```suggestion
        exception_data = raw_data["exception"]
        extra = {
            "message": exception_data["message"],
            "cause": exception_data["cause"],
            "severity": enums.ExceptionSeverity(exception_data["message"]),
        }
```

Reply:
This would cause issues for users who haven't updated their server to a 3.4 jar as the exception key won't be there for them.

Reply:
That doesn't matter, users are expected to be running the version newer or equal to the version that is shipped with Red.

Reply:
Red-Lavalink is theoretically not locked to Red alone.

Reply:
True but 0.x.0 versions can be breaking *and* we've bumped minimal jar version in the past:
#71